.github/workflows/ci.yml

```diff
@@ -351,17 +226,6 @@ jobs:
         with:
           name: coverage-regular-tests
           path: coverage-data
-      - name: Download Coverage Reports
-        uses: actions/download-artifact@v4
-        with:
-          name: coverage-cmis-tests
-          path: coverage-data
-      - name: Combine reports
-        run: |
-          pip install $(grep codecov= requirements/ci.txt)
-          ls -la coverage-data
-          coverage combine coverage-data/.coverage-regular coverage-data/.coverage-cmis
-          coverage report
 
       - name: Publish coverage report
```

Comment:
Is the report still properly uploaded? Since I don't see a comment on the PR from codecov currently :thinking: